tools/blobtoolkit/macros.xml

```diff
@@ -42,6 +42,8 @@
             </when>
             <when value="disabled"/>
         </conditional>
-
+    </xml>
+    <xml name="macro_taxdump">
+        <param argument="--taxdump" type="data" format="tar" label="NCBI taxdump directory" help="Please, check that the datatype is set as tar; during the upload process, Galaxy decompress it automatically. You can find more information about it in the help section."/>
```

Comment:
Is that really tar data type or tar.gz?

Reply:
Indeed it also works even if Galaxy recognizes it as gz instead of tar, so I don't think it is a problem after all.

Reply:
The main requirement is that it should be the last taxdump version.